RecoLocalTracker/SiPixelRecHits/src/PixelCPEGeneric.cc

```diff
@@ -113,26 +113,26 @@ PixelCPEGeneric::PixelCPEGeneric(edm::ParameterSet const & conf,
     xerr_endcap_def_=0.0020;
     yerr_endcap_= {0.00210};
     yerr_endcap_def_=0.00075;
-  } else { // isUpgrade=true, funny case
-    xerr_barrel_ln_= {0.00114,0.00104,0.00214};
-    xerr_barrel_ln_def_=0.00425;
-    yerr_barrel_ln_= {0.00299,0.00203,0.0023,0.00237,0.00233,0.00243,0.00232,0.00259,0.00176};
-    yerr_barrel_ln_def_=0.00245;
-    xerr_endcap_= {0.00151,0.000813,0.00221};
-    xerr_endcap_def_=0.00218;
-    yerr_endcap_= {0.00261,0.00107,0.00264};
-    yerr_endcap_def_=0.00357;
+  } else { // isUpgrade=true    
+    xerr_barrel_ln_= {0.00025, 0.00030, 0.00035, 0.00035};
```

Comment:
First of all thanks for spotting the missing "0" in default value for xerr_barrel_def

I share your interpretation that nrow=1 population in the barrel is very low because of the Lorentz angle drift but right now I do not have a map of the simhit position within a pixel cell for this category of events supporting this hypothesis. On the other hand I see no bias in the simhit-rechit residuals or a drop of the efficiency (at least in the SingleMuPt10 events) so I would not inflate the error unless really see dramatic effects in the relval.